app_dart/test/request_handlers/check_for_waiting_pull_requests_test.dart

```diff
@@ -122,6 +122,8 @@ void main() {
       );
     }
 
+    // TODO(fujino): Uncomment once https://github.com/flutter/flutter/issues/68632 is resolved
+    /*
```

Comment:
It was non-trivial to make these `//` style comments, because of the multi-line strings.

Reply:
Can we skip this test instead, and link to the issue?

Reply:
SGTM